peer/brontide.go

```diff
@@ -82,6 +83,10 @@ const (
 	// MaxPongBytes (upper bound accepted by the protocol) because it is
 	// needlessly wasteful of precious Tor bandwidth for little to no gain.
 	PongSizeCeiling = 4096
+
+	// TorTimeoutMultiplier is the scaling factor we use on network timeouts
+	// for Tor peers.
+	TorTimeoutMultiplier = 3
```

Comment:
q: Is there a reason we have to export this?

Reply:
Nah, we can (and should) keep private.